src/ActionModal/components/ActionModal.vue

```diff
@@ -139,6 +142,19 @@
           </TmFormGroup>
         </form>
       </div>
+      <div v-else-if="step === `send`" class="action-modal-form">
+        <TmDataMsg icon="hourglass_empty">
+          <div slot="title">
+            Sent and confirming
+          </div>
+          <div slot="subtitle">
+            The transaction
+            <!--with the hash {{ txHash }}-->
```

Comment:
Is this meant to be commented still?

Reply:
Left in there to signal possible improvements. I can remove it.

Reply:
It's fine 👍